archaius2-core/src/main/java/com/netflix/archaius/DefaultConfigLoader.java

```diff
@@ -70,7 +70,13 @@ public Builder withConfigReader(ConfigReader loader) {
             this.loaders.add(loader);
             return this;
         }
-        
+
+        public Builder withConfigReaders(Set<ConfigReader> loaders) {
```

Comment:
Why does this take a `Set`? Since `this.loaders` is a List multiple calls can still result in duplicates.